### DISCUSSION TITLE
Intersecting text creates clutter in the image

### DISCUSSION THREAD

**Discussion by Gushchin24** [ogkalu2/comic-translate#52] [General]
I don't understand how this can be adjusted manually. When a translation is generated, the text fields to be translated are automatically detected and often appear incorrectly in the final result.

**Comment by Gushchin24** [comment]
![2018-03-20-Tue](https://github.com/ogkalu2/comic-translate/assets/11369565/84f1ad2e-00fb-43aa-8b74-bb0cc87488f1)

**Reply by TaunT** [reply]
The same problem)

**Comment by Gushchin24** [comment]
![2018-03-20-Tue_translated](https://github.com/ogkalu2/comic-translate/assets/11369565/514142d6-abba-48fb-820a-0099ece66f13)
